tests/helper_functions.py

```diff
@@ -552,10 +552,7 @@ def assert_array_almost_equal(actual_array, desired_array, rtol=1e-6):
 
 
 def _mlflow_major_version_string():
-    ver = Version(mlflow.version.VERSION)
-    major = ver.major
-    minor = ver.minor
-    return f"mlflow=={major}.{minor}"
+    return _generate_mlflow_version_pinning()
```

Comment:
`_mlflow_major_version_string` needs a fix. It never returns a major version. I'll fix this in a follow-up.